### PR TITLE
feat(ttl): add support for track

### PR DESCRIPTION
**Pull request by reversedClaustrum**

### CODE REVIEW

**Comment by reversedClaustrum** [comment]
ttltrack

**Comment by su060805** [comment]
@reversedClaustrum Hi, what's the meaning of the directory `.idea`? Anything affects?

**Review by su060805 — CHANGES_REQUESTED** [review]
Could you please remove the useless codes and perhaps using english code comments for internationalizing?

**Comment by su060805** [comment]
I will add a ci for checking pr titles later, and could @Cypress0522 please setup a Rust unit test workflow? Also, I think @reversedClaustrum should add some unit tests to make your codes works properly.

File: src/ttltrack.rs [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+const TCP_CONNRECORD_KEY_LEN: usize = 37; // TCP连接记录键的长度
+const TCP_CLEANUP_INTERVAL_SEC: u64 = 30; // 清理间隔时间（秒）
+
+// TCP连接跟踪信息结构体
+#[derive(Clone)]
+pub struct TcpConntrackInfo {
+    pub is_ipv6: u8, // 是否为IPv6
+    pub ttl: u8, // 生存时间
+    pub srcip: [u32; 4], // 源IP地址
+    pub srcport: u16, // 源端口
```
Comment (su060805) [inline src/ttltrack.rs]: Please use semantic styled naming conventions like `src_port` and `src_ip`, I'll setup cspell and it's workflow later.

**Review by su060805 — CHANGES_REQUESTED** [review]

File: src/ttltrack.rs [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+            addr,
+            packet,
+            packet_len,
+            packet_v6,
+            ttl_of_fake_packet,
+            do_wrong_chksum,
+            do_wrong_seq,
+        );
+    }
+}
```
Comment (su060805) [inline src/ttltrack.rs]: Perhaps add unit tests for all functions.

**Pull request closed without merging**